docs/source/FAQ.rst

```diff
@@ -145,3 +145,31 @@ The issue can be fixed in two steps:
    free the PVs and you can continue with the testing but please make sure that
    you can afford to lose the data stored on the PVs you are about the free.
 
+
+**7. How can I build test-operator-index locally?**
```

Comment:
One more thing please :see_no_evil: :smile: . Can you modify this [1] line in the documentation:
```
ENABLE_WEBHOOKS=false make install run
```

It'll make the installation easier for lot of people (most of the people won't need to modify the webhooks during development). We can extend the documentation later.

[1] https://github.com/openstack-k8s-operators/test-operator/blob/main/docs/source/guide.rst#running-test-operator-locally-outside-the-cluster

Reply:
I have added the changes, please check if they are as you requested :smile:

Reply:
The changes look good:) ... Thank you!